pyproject.toml

```diff
@@ -25,6 +25,10 @@ archs = ["x86_64", "arm64"]
 [tool.pytest.ini_options]
 addopts = "-ra -v"
 filterwarnings = [
```

Comment:
In `scippnexus` I just added `errors` here, to convert warnings to errors, unless explicitly ignored.

Reply:
There are a lot of warnings in scipp. We should address all that we can and silence the others. But a lot of them are plotting related and I am not sure we should chase them down at this point.